cmd/clusterctl/client/cluster/mover.go

```diff
@@ -333,6 +335,19 @@ func (o *objectMover) move(graph *objectGraph, toProxy Proxy, mutators ...Resour
 		return errors.Wrap(err, "error pausing ClusterClasses")
 	}
 
+	log.V(2).Info("Waiting for all resources to be ready to move")
```

Comment:
```suggestion
	log.Info("Waiting for all resources to be ready to move")
```
Proposing a higher verbosity so the user gets a stronger signal that the system is waiting for something